code/datums/mutations/chameleon.dm

```diff
@@ -57,3 +57,6 @@
 		return
 	owner.alpha = 255
 	UnregisterSignal(owner, list(COMSIG_MOVABLE_MOVED, COMSIG_HUMAN_EARLY_UNARMED_ATTACK))
+	/// SKYRAT EDIT BEGIN
+	REMOVE_TRAIT(owner, TRAIT_CHAMELEON_SKIN, GENETIC_MUTATION)
```

Comment:
Sounds like that should probably be taken upstream, try to reproduce it there and just make the PR there if you can instead. That means less code for us to maintain, and allows everyone to benefit from the fix.

Reply:
Good job Golden you didn't even take the time to read the comments and actually check the code too hard.

That's what I get for reviewing when I should be asleep.